packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -184,6 +186,9 @@ class _DismissibleState extends State<Dismissible> with TickerProviderStateMixin
   Size _sizePriorToCollapse;
 
   @override
+  bool get wantKeepAlive => _moveController?.isAnimating == true || _resizeController?.isAnimating == true;
```

Comment:
If you want to get fancy, it could be your ticker provider too since that will let it get all the animation controllers registered very succinctly.